spec/clients/cornershop_client_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe CornershopClient do
+  let(:query) { 'pan' }
+  let(:mocked_response) do
+    {
+      "results":
+      [
+        {
+          "store": {
+            "id": 4,
+            "name": "Jumbo"
+          },
+          "search_result":
+            {
+              "search_term": "pan",
+              "aisles": [
+                {
+                  "aisle_id": "C_572",
+                  "products":
+                  [
+                    {
+                      "id": 376521,
+                      "brand": { "id": 1893, "name": "Kingsbury" },
+                      "kind": "PRODUCT",
+                      "related_to": nil,
+                      "buy_unit": "UN",
+                      "default_buy_unit": "UN",
+                      "purchasable": true,
+                      "price": 1950.0,
+                      "original_price": nil,
+                      "name": "Pan molde perfecto blanco",
+                      "package": "Bolsa 600 g",
+                      "img_url": "https://s.cornershopapp.com/img-url"
+                    },
+                    {
+                      "id": 376522,
+                      "brand": { "id": 1894, "name": "Ideal" },
+                      "kind": "PRODUCT",
+                      "related_to": nil,
+                      "buy_unit": "UN",
+                      "default_buy_unit": "UN",
+                      "purchasable": true,
+                      "price": 1950.0,
+                      "original_price": 2250.0,
+                      "name": "Pan molde XL",
+                      "package": "Bolsa 10 rebanadas",
+                      "img_url": "https://s.cornershopapp.com/img-url-2"
+                    }
+                  ]
+                }
+              ]
+            }
+        }
+      ]
+    }
+  end
+
+  let(:expected_response) do
+    [
+      {
+        provider: hash_including(name: 'Jumbo'),
+        products:
+          [
+            {
+              price: 1950.0,
+              measure: "UN",
+              package: "Bolsa 600 g",
+              name: "Kingsbury - Pan molde perfecto blanco",
+              img_url: "https://s.cornershopapp.com/img-url"
+            },
+            {
+              price: 2250.0,
+              measure: "UN",
+              package: "Bolsa 10 rebanadas",
+              name: "Ideal - Pan molde XL",
+              img_url: "https://s.cornershopapp.com/img-url-2"
+            }
+          ]
+        }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstHashElementIndentation: Indent the right brace the same as the start of the line where the left brace is.